### PR TITLE
exclude joi modules and skip invalid semver versions

### PR DESCRIPTION
**Pull request by lloydbenson**

### CODE REVIEW

**Review by Nargonath — APPROVED** [review]

**Comment by Nargonath** [comment]
Thanks @lloydbenson

File: assets/js/getModuleInfo.js [hunk 1]
```diff
@@ -71,15 +71,21 @@ const hapiPlugins = [
...
+  "formula",
   "good",
   "good-console",
   "good-squeeze",
   "hapi.dev",
   "items",
+  "joi",
+  "joi-date",
+  "joi.dev",
+  "pinpoint",
```
Comment (Marsup) [inline assets/js/getModuleInfo.js:88]: Shouldn't you exclude tlds as well?

**Comment by lloydbenson** [comment]
interestingly I didn't see any errors around this, but yeah we can add that too. I'll make another PR.

**Comment by lloydbenson** [comment]
https://github.com/hapijs/hapi.dev/pull/520

**Pull request merged**